third_party/llvm-project/libunwind/src/RWMutex.hpp

```diff
@@ -14,7 +14,7 @@
 #define __RWMUTEX_HPP__
 
 #if defined(_LIBUNWIND_HAS_STARBOARD_THREADS)
-#include "starboard/common/rwlock.h"
+#include <shared_mutex>
```

Comment:
This lgtm as is, and lets merge it. As a follow-up, i'd like to try if we can actually remove Starboard block from here entirely, and just use the raw/default `pthreads` code below. E.g. `pthread_rwlock_rdlock` calling options

Reply:
I've tried to do so, didn't landed from the first attempt - it breaks evergreen build. I'll try to figure it out.

Reply:
Apparently, evergreen build uses pthread from musl lib (third_party/musl/src/starboard/pthread/pthread.h), that does not support pthread_rwlock_t.